### PR TITLE
Add organization member: @zdroid

### LINKED ISSUE #3663: Add organization member: @zdroid (more shameless self-promotion)
It seems #3644 was my 10th PR merged, not sure which was the 5th one I reviewed.

Note: it also seems @mebeim has been an active organization member for more than 6 months. Shouldn't he become an organization owner now?

### PR DESCRIPTION
**Pull request by zdroid**
Closes #3663.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ If you are an owner of the organization, you should be able to see an automated
   [19 November 2019](https://github.com/tldr-pages/tldr/issues/3580) - present
 - **Proscream ([@Proscream](https://github.com/Proscream))**:
   [22 November 2019](https://github.com/tldr-pages/tldr/issues/3592) - present
-- **Zlatan Vasović ([@zdroid](https://github.com/zdroid))**:
-  [30 November 2019](https://github.com/tldr-pages/tldr/issues/3636) - present
+- Zlatan Vasović ([@zdroid](https://github.com/zdroid)):
+  [30 November 2019](https://github.com/tldr-pages/tldr/issues/3636) - [17 December 2019](https://github.com/tldr-pages/tldr/issues/3663)
```
Comment (mebeim) [inline MAINTAINERS.md]:
```suggestion
```
We usually *remove* people from the "collaborators" list when they become members. Please remove this line.

Comment (agnivade) [reply MAINTAINERS.md]: Oh yeah, good point. I always forget these. Nice catch.

Comment (zdroid) [reply MAINTAINERS.md]: Thanks for suggestion, I wasn't sure about this one.

Comment (waldyrious) [reply MAINTAINERS.md]: Wouldn't it be better to keep the line for future record, though? Otherwise it will be unnecessarily hard to track down.

Comment (waldyrious) [reply MAINTAINERS.md]: By the way, I think last time discussed this, the conclusion was to use bold to denote current members of a group, and plain formatting for the former ones. That's already the case for the organization members and owners section of this document, so it's only consistent to apply the same logic here as well.

**Comment by zdroid** [comment]
@waldyrious Okay, I reverted it. Who is supposed to give the final say, @sbrl maybe?

**Comment by waldyrious** [comment]
I'd say this should only be closed once the invite is sent out and accepted. Any of us with merge permissions can merge once we can verify that you're part of the org (remember that your membership should be public).

**Comment by sbrl** [comment]
Hey! Sorry, I've been busy with wrapping up the end of my first PhD semester. Sending invite now :-)

**Comment by zdroid** [comment]
All set, thanks to everyone!

**Pull request merged**